scripts/combine_zarr.py

```diff
@@ -112,13 +119,14 @@ def setup_output(output_path, seqid, field, example_arr, samples, cname, clevel,
     log('Setting up output at {!r} ...'.format(output_path))
     callset = zarr.open_group(output_path, mode='a')
     seq_group = callset.require_group(seqid)
-    calldata_group = seq_group.require_group('calldata')
+    field_root, field_id = field.split("/")
```

Comment:
Should I default to `calldata` here, irrespective of whether the original comes from `variants` or not?

Reply:
Yes I think always use `calldata` regardless of whether original comes from `calldata` or `variants`.